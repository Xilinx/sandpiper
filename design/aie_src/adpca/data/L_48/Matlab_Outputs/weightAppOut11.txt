2.746902847e+01 1.172524719e+02
-4.433631897e-01 3.113945007e+00
-6.036757469e+00 6.593096733e+00
