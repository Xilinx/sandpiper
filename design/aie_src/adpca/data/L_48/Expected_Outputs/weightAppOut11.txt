2.746557999e+01 1.172483902e+02 
-4.434874058e-01 3.114022255e+00 
-6.036589146e+00 6.592769623e+00 
